### PR TITLE
feat: added create contact us message

### PR DESCRIPTION
**Pull request by mmubarak0**
Created the endpoint `/api/v1/contact`

## description
This endpoint is designed for the submission of 'Contact Us' messages.

## Related issue
- #26
    - https://github.com/hngprojects/hng_boilerplate_nestjs/issues/101
- possible duplicate of 26 / #302

## Motivation and Context
Every organization requires a dedicated point of contact for inquiries, leading to the creation of the 'Contact Us' section. This essential feature serves as a bridge between the organization and the public, facilitating communication and information exchange. It's a fundamental aspect of customer service and user engagement, ensuring that questions, feedback, and support requests are directed to the appropriate channels for prompt and efficient resolution.

## How Has This Been Tested?
It has been tested using pytest and curl command.

## Screenshots (if appropriate - Postman, etc):
image
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x]  My code follows the code style of this project.
- [x]  My change requires a change to the documentation.
- [x]  I have updated the documentation accordingly.
- [x]  I have read the CONTRIBUTING document.
- [x]  I have added tests to cover my changes.
- [x]  All new and existing tests passed.

### CODE REVIEW

File: api/v1/routes/contact_us.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+contact_us = APIRouter(tags=["Contact Us"])
+
+
+# CREATE
+@contact_us.post("/contact-us", response_model=JsonResponseDict.response)
+async def create_contact_us(data: CreateContactUs, db: Annotated[Session, Depends(get_db)]):
+    """Add a new contact us message."""
+    new_contact_us_message = contact_service.create(db, data)
+    response = JsonResponseDict(
```
Comment (joboy-dev) [inline api/v1/routes/contact_us.py:19]: Please use success_response() function and I will merge

**Review by joboy-dev — COMMENTED** [review]
Make changes

**Comment by joboy-dev** [comment]
Merge conflict!!

**Comment by mmubarak0** [comment]
Review this also after you done this
 - #523

**Comment by mmubarak0** [comment]
🫠 again that same conflict is happening on every merge am tired. tell me when you want you to review my pr then I will fix it

**Review by Goketech — APPROVED** [review]

**Review by joboy-dev — APPROVED** [review]

**Review by SundayMba — APPROVED** [review]

**Pull request merged**